app/models/user.rb

```diff
@@ -0,0 +1,13 @@
+class User < ActiveRecord::Base
+
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
+
+  validates :username, :presence => true,
+            :uniqueness => {:case_sensitive => false}
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.